Microsoft.VisualStudio.CodeTools/CodeToolsSetup/buildMSM.xml

```diff
@@ -55,11 +55,16 @@
     <Platform>Mixed Platforms</Platform>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(VisualStudioVersion)'==''">
+    <VisualStudioVersion Condition="'$(MSBuildToolsVersion)'=='14.0'">14.0</VisualStudioVersion>
+    <VisualStudioVersion Condition="'$(MSBuildToolsVersion)'!='14.0'">12.0</VisualStudioVersion>
```

Comment:
This changes the current behavior which is <PlatformToolset>v110</PlatformToolset>